okio/src/appleTest/kotlin/okio/AppleByteStringTest.kt

```diff
@@ -28,4 +28,10 @@ class AppleByteStringTest {
     val byteString = data.toByteString()
     assertEquals("Hello", byteString.utf8())
   }
+
+  @Test fun emptyNsDataToByteString() {
+    val data = ("" as NSString).dataUsingEncoding(NSUTF8StringEncoding) as NSData
+    val byteString = data.toByteString()
+    assertEquals("", byteString.utf8())
```

Comment:
Does `assertEquals(ByteString.EMPTY, byteString)` work here?

Reply:
Was doing like the first test which was asserting with string. Will change.